src/st_stuff.h

```diff
@@ -66,6 +66,13 @@ extern int health_red;    // health amount less than which status is red
 extern int health_yellow; // health amount less than which status is yellow
 extern int health_green;  // health amount above is blue, below is green
 
+static inline boolean ST_PlayerInvulnerable(struct player_s *player)
```

Comment:
Please use `player_t` instead of `struct player_s` here.

Reply:
Does it make a difference apart from coding style?

Reply:
clang-tidy is complaining about this.